paasta_tools/setup_tron_namespace.py

```diff
@@ -97,6 +97,26 @@ def ensure_service_accounts(job_configs: List[TronJobConfig]) -> None:
                         namespace=spark_tools.SPARK_EXECUTOR_NAMESPACE,
                         kube_client=spark_kube_client,
                     )
+            elif (
```

Comment:
Do we need this block since there is always an iam_role returned by `action.get_iam_role()` if `action.get_executor() == "spark"`, so we will never hit this condition.
If the objective is to safeguard against `action.get_iam_role()` being None, then we can just take out the spark executor service account creation logic from this if else block and put it under `if action.get_executor()=="spark" and action.get_spark_executor_iam_role()`
What do you think?

Reply:
ah, you're right: this should just be `action.get_executor() == "spark" and action.get_spark_executor_iam_role()`!

Reply:
ah, i see - we can just combine the previous logic and this and call ensure_service_account twice: once with get_iam_role (for the driver) and then with get_spark_executor_iam_role for the executor